assets/css/login.less

```diff
@@ -16,50 +16,65 @@ body {
     top: 80px;
     left: 300px;
     width: 300px;
-    height: 250px;
-    border: none;
-    border-radius: 3px;
+    height: 250px;    
+}
+
+.rounded-border {
+    border-radius: 3px; 
 }
 
-.skin_login {
+.skin-login {
      background-color: white;
-     box-shadow: 0px 2px 2px #333435;   
+     box-shadow: 0px 2px 2px #333435; 
+     border: none;
 }
 
-.login_header {    
+.login-header {    
     position: absolute; 
-    width:100%;
-    border-radius: 3px;
+    width:100%;    
     padding-top: 25px;
-    text-align: center;
-    vertical-align: middle;
     line-height: 20px;
     height: 40px;
 }
 
-.skin_login_header {
+.text-center {
+    text-align: center;
+    vertical-align: middle;
+}
+
+.skin-login-header {
     background-color: #419fe2; 
     color: white;
     font-size: 20px;
 }
 
-.login_input {
-    border: none;
-    border-bottom: 0.5px solid grey;
+.login-input {   
     width: 80%;
     margin-left: 30px;
     margin-right: 10px;
 }
 
-.login_submit {    
+.login-input-1 {
+    margin-top: 90px;
+}
+
+.login-input-2 {
+    margin-bottom: 45px;
+}
+
+.skin-login-input {
+    border: none;
+    border-bottom: 0.5px solid grey;
```

Comment:
why 0.5 px for a border width? :)

Reply:
I was trying to make it thin, so that it would look as similar as possible to the original page in the pdf we got.

Reply:
are you sure the PDF file was not zoomed out? I am pretty sure Vali did not put a .5px border in the mocks. It is safe to make it 1px